doc/JSON_INFO.md

```diff
@@ -711,6 +711,7 @@ Vehicle components when installed on a vehicle.
                                *    power       = base engine power in watts
                                *    bonus       = bonus granted; muffler = noise reduction%, seatbelt = bonus to not being thrown from vehicle
                                *    par1        = generic value used for unique bonuses, like the headlight's light intensity */
+"cargo_modifier": 33,         // (Optional, default = 100) Modifies cargo weight by set percentage
```

Comment:
So we're defining that a seperate dimension has mass in the current dimenion?

Reply:
It's general modifier. Current other dimension is set to "cargo_modifier": 0